### PR TITLE
Update calling-functions.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/python-functions/calling-functions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-functions/calling-functions.md?ref=a661914a4d04157441de7dadc194f5a94b9f281b)

File: python/python-core/python-functions/calling-functions.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Gives the following output:
 Function time!
 ```
 
-A parameter in Python is a variable which can be passed into functions so that methods can be performed onto it.
+A parameter in Python is a variable that can be passed into functions so that methods can be performed onto it.
```
Comment (nem035) [inline python/python-core/python-functions/calling-functions.md:43]: We can do more than methods on any value so I'd maybe remove that part? Also, parameters aren't technically passed in, it's values for those parameters. i.e. the arguments, that are passed in. 

```suggestion
A parameter in Python is a variable that represents a value passed into a function.
```

File: python/python-core/python-functions/calling-functions.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Gives the following output:
...
 
 In this example, variable `x` is passed into the function as a parameter:
```
Comment (nem035) [inline python/python-core/python-functions/calling-functions.md:45]:
```suggestion
In this example, variable `x` is defined in the function as a parameter:
```

File: python/python-core/python-functions/calling-functions.md [hunk 3]
```diff
@@ -50,14 +50,18 @@ def func(x):
     print(ans)
 ```
 
-So to call this function, you must pass a parameter into your function call as follows:
+To call this function, you must pass a parameter into it. You can pass a number directly or through a variable:
```
Comment (nem035) [inline python/python-core/python-functions/calling-functions.md:53]:
```suggestion
To call this function, you must pass a value (i.e. an argument) into it, which will put that value into the `x` variable (i.e. parameter) inside the function.

You can pass a number directly or through a variable:
```

**Review by nem035 — DISMISSED** [review]

**Review by kapnobatai137 — APPROVED** [review]
:rocket:

**Pull request merged**